lib/wasi/src/ptr.rs

```diff
@@ -83,9 +83,4 @@ impl<T: Copy + ValueType> WasmPtr<T, Array> {
     pub unsafe fn get_utf8_str(self, memory: &Memory, str_len: u32) -> Option<&str> {
         self.0.get_utf8_str(memory, str_len)
     }
-
-    #[inline(always)]
-    pub fn get_utf8_string(self, memory: &Memory, str_len: u32) -> Option<String> {
-        self.0.get_utf8_string(memory, str_len)
-    }
```

Comment:
I think we want to undelete this as I believe it's a publicly visible API. I'm not sure to what extent it's useful as this is the WASI WasmPtr, but if it's publicly visible, we probably shouldn't remove it